validator-api/validator-api-requests/src/models.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, PartialEq, Eq)]
+#[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
```

Comment:
Is the feature = `ts-rs` correct still?

Reply:
Correct. If you notice the `typescript-types` was defined for the client crate and this refers to the `validator-api-requests` crate. This crate was introduced to remove duplication of type definitions. And as for the new "feature", it was defined due to name clash of `ts-rs`